Parent/Parent/Routes.swift

```diff
@@ -56,7 +56,7 @@ let router = Router(routes: [
         guard let courseID = params["courseID"], let assignmentID = params["assignmentID"] else { return nil }
         guard let session = Session.current, let studentID = currentStudentID else { return nil }
         if assignmentID == "syllabus" {
-            return CourseSyllabusViewController(courseID: courseID, studentID: studentID, session: session)
+            return CourseDetailsViewController.create(courseID: courseID, studentID: studentID)
```

Comment:
This introduces a bug: if the course syllabus is not the front page and is not visible in the tabs, a teacher might still link to the syllabus. With this change it won't be shown (because of the logic in `CourseDetailsViewController`). I'm only aware of this case because Android had this very bug and had a ticket come through support for it.

I think we should return the standalone syllabus for this explicit route. Is that doable?

Reply:
That is very doable.